src/Markdown.js

```diff
@@ -56,23 +57,47 @@ export default class Markdown {
         return is_plain;
     }
 
-    toHTML() {
+    render(html) {
```

Comment:
What do you want me to rename it to?  `toHTML` isn't really an apt name anymore with this.